### DISCUSSION TITLE
In Person meeting without any address on website

### DISCUSSION THREAD

**Discussion by bobafett0** [code4recovery/12-step-meeting-list#1038] [Ideas]
In our recovery program we have a use case for wanting a meeting's time, city, and state listed on our site, but want them to reach out to a coordinator in order to get the exact address.

For in-person meetings, the plugin doesn't seem to allow me to post a change to the meeting unless it has an address and exact location. 
So far in the "Location" and "Notes" we put something like "email us for location", and then put in a false address, which has confused people and thrown them off.

It'd be nice to have the option for posting in-person meetings without being required to supply an address, i.e. exact location.

**Comment by joshreisner** [comment]
Yes, sorry, it's always been a firm requirement of this plugin that a valid street address must be provided for a meeting to be listed.

Getting offices to provide standardized addresses that are presented to the user with maps, distances, and directions is pretty much the main reason this plugin exists.

Only in the age of Covid did we start to allow approximate addresses for online meetings.

If your program doesn't want geolocation services, a simpler approach would be to use a table plugin like [wpDataTables](https://wpdatatables.com/)
